yt_dlp/extractor/indavideo.py

```diff
@@ -78,13 +73,12 @@ def _real_extract(self, url):
         for video_url in video_urls:
             height = int_or_none(self._search_regex(
                 r'\.(\d{3,4})\.mp4(?:\?|$)', video_url, 'height', default=None))
-            if filesh:
-                if not height:
-                    continue
-                token = filesh.get(compat_str(height))
-                if token is None:
-                    continue
-                video_url = update_url_query(video_url, {'token': token})
```

Comment:
The old code implies that the `token` is optional and `video['filesh']` may be non-existent, and as such the old token code was completely non-fatal. Do you know for certain that every video requires a token now?

Reply:
> The old code implies that the `token` is optional and `video['filesh']` may be non-existent, and as such the old token code was completely non-fatal. Do you know for certain that every video requires a token now?

My experience is that all video links requires token for a long time.